### ISSUE TITLE
Working with Onvif 2.0

### ISSUE THREAD

**Issue by diasrobson** [45clouds/homebridge-ip-camera#9]
I have lots of IP cameras around the house and I was wondering if that work with this plug in here is a link to them
 
https://domar.com/atlas-ii-poe-2mp-30m-nightvision-varifocal-lens-cctv-dome-ip-camera-mcidp17.html

**Comment by 45clouds** [comment]
Yes. These cameras support RTSP so it will be ease to integrate with them. Just use default example and provide correct IP address of your cameras.
